package/src/index.ts

```diff
@@ -338,23 +338,27 @@ export default function <ThemeName extends string, Schema extends z.ZodTypeAny>(
 					};
 
 					// Initialize route injection
-					const { pages, injectPages } = addPageDir(pageDirOption);
+					const { pages: pagesInjected, injectPages } = addPageDir(pageDirOption);
+					const pagesResolved: Record<string, string | false> = Object.fromEntries(
+						Object.keys(pagesInjected).map((p) => [p, p]),
```

Comment:
I'd love these `p` to be something like `pageKey`

Reply:
Good call!